lib/web_ui/test/ui/image_golden_test.dart

```diff
@@ -106,14 +104,19 @@ Future<void> testMain() async {
 
         final ui.PictureRecorder recorder = ui.PictureRecorder();
         final ui.Canvas canvas = ui.Canvas(recorder, drawRegion);
-        canvas.drawImage(image, ui.Offset.zero, ui.Paint()..filterQuality = ui.FilterQuality.none);
-        canvas.drawImage(image, const ui.Offset(150, 0), ui.Paint()..filterQuality = ui.FilterQuality.low);
-        canvas.drawImage(image, const ui.Offset(0, 150), ui.Paint()..filterQuality = ui.FilterQuality.medium);
-        canvas.drawImage(image, const ui.Offset(150, 150), ui.Paint()..filterQuality = ui.FilterQuality.high);
+        canvas.drawImage(image, ui.Offset.zero,
```

Comment:
There are a ton of changes in this file that are just random whitespace differences. To avoid churn, can you try to revert the changes that are unrelated to the test you added?

Reply:
Yeah I was kind of fighting the default dart format here. Does the engine have a set of format options or is everyone doing it manual?

Reply:
We're (thankfully) working on getting formatting rolling into the engine (and hopefully flutter/flutter) after not too long here. Send happy vibes to @matanlurey who is driving!

Reply:
...at the moment it's all by hand (in my opinion, silliness)